src/pages/settings/company/common/hooks/useCompanyDetailsTabs.tsx

```diff
@@ -0,0 +1,40 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { route } from 'common/helpers/route';
+import { Tab } from 'components/Tabs';
+import { useTranslation } from 'react-i18next';
+
+export function useCompanyDetailsTabs() {
+  const { t } = useTranslation();
+
+  const tabs: Tab[] = [
+    { name: t('details'), href: route('/settings/company_details/details') },
```

Comment:
This should be the default route. So `href` should be`/settings/company_details`. I believe at the moment it won't highlight the first tab cause of a non-existing route, right?

Reply:
> 

Yeah sure, I missed that, now it's changed.